src/formatDate.js

```diff
@@ -8,7 +8,75 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  // create property for symbols from and to
+  const simbolFormat = fromFormat[fromFormat.length - 1];
+  const simbolTo = toFormat[toFormat.length - 1];
+  // create split array from date
+  const dateDay = date.split(simbolFormat);
```

Comment:
The variable 'dateDay' is misleadingly named as it suggests it contains only the day part of the date. A name like 'dateParts' would better represent its contents, which are the parts of the date split by the format symbol.